test/routable_component.js

```diff
@@ -63,75 +63,75 @@ describe('RoutableComponent', function() {
         );
         this.MyEventSourceComponent.emit('myevent', "1", 2),
         assert(this.subject.controllers.MyController.component === this.subject);
-        assert.deepEqual(this.subject.controllers.MyController.myaction_args, ["1", 2]);
+        assert.deepEqual(this.subject.controllers.MyController.myactionArgs, ["1", 2]);
       });
     });
   });
 
-  /** @test {RoutableComponent#register_component} */
-  context('register_component', function() {
-    subject(function() { return new RoutableComponent({}, this.routes, this.controller_classes) });
+  /** @test {RoutableComponent#registerComponent} */
+  context('registerComponent', function() {
+    subject(function() { return new RoutableComponent({}, this.routes, this.controllerClasses) });
 
     it('surely registered', function() {
-      this.subject.register_component('MyEventSourceComponent', new MyEventSourceComponent());
+      this.subject.registerComponent('MyEventSourceComponent', new MyEventSourceComponent());
```

Comment:
__[eslint]__
* Error - Unexpected 'this'. (no-invalid-this)

<!-- Comment by [SideCI](https://sideci.com) -->